bubble_sort/Java/BubbleSortRecursive.java

```diff
@@ -0,0 +1,46 @@
+public class BubbleSortRecursive {
+    public static void main(String[] args){
+    
+    //Taking a sample array for sorting it.
+        int[] nums = {12, 45, 23, 78, 67};
+
+    //calling the method and passing the array in it.
+        bubble(nums, nums.length);
+	
+    //Printing the sorted array
+        printSortedArray(nums);
+    }
+    
+    //The recurisve method that takes in the array and the length of it.
+    public static void bubble(int[] nums, int end){
+	
+    //base case
+        if (end == 1) {
+            return;
+        }
+	
+    //for loop that iterates and sorts the sorts the array
+        for (int i = 0; i < end - 1; i++) {
+            if (nums[i] > nums[i + 1]) {
+                int temp = nums[i];
+                nums[i] = nums[i + 1];
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpddmzpmr6/bubble_sort/Java/BubbleSortRecursive.java
+++ b/tmp/tmpddmzpmr6/bubble_sort/Java/BubbleSortRecursive.java
@@ -6,7 +6,7 @@
 
     //calling the method and passing the array in it.
         bubble(nums, nums.length);
-	
+
     //Printing the sorted array
         printSortedArray(nums);
     }
```